### PR TITLE
Clearify how to use Getters & Setters

### PR DESCRIPTION
**Pull request by phanect**
There were no concrete example of getters and setters in "Defining as part of a property" section, so I rewrite it.

### CODE REVIEW

File: docs/docs/models.md [hunk 1]
```diff
@@ -134,22 +134,30 @@ Getters and Setters can be defined in 2 ways &lpar;you can mix and match these 2
...
+    set      : function(val) {
+      this.setDataValue('title', val.toUpperCase());
+    }
   }
 });
+
+Employee
+  .create({ name: 'John Doe', title: 'senior engineer' })
+  .then(function(employee) {
+    console.log(employee.name); // John Doe (SENIOR ENGINEER)
```
Comment (mickhansen) [inline docs/docs/models.md]: could you change these two to `employee.get('name')` and `employee.get('title')` respectively please.

File: docs/docs/models.md [hunk 2]
```diff
@@ -134,22 +134,30 @@ Getters and Setters can be defined in 2 ways &lpar;you can mix and match these 2
...
     type     : Sequelize.STRING,
     allowNull: false,
     get      : function()  {
-       /*
-         do your magic here and return something!
-         'this' allows you to access attributes of the model.
- 
-        example: this.getDataValue('name') works
-      */
+      var title = this.getDataValue('title'); // 'this' allows you to access attributes of the model.
```
Comment (mickhansen) [inline docs/docs/models.md]: "model" should probably be "instance" here.

File: docs/docs/models.md (hunk 1 above)
Comment (phanect) [inline docs/docs/models.md]: Sorry, I didn't your intention. Is `employee.get('name')` more recommended way to get values than `employee.name`?
In my understanding `employee.name` is more popular way...

Comment (mickhansen) [inline docs/docs/models.md]: `employee.name` is sort of deprecated, it maps to `employee.get('name')` internally and introduces some complexity and overhead with `Object.defineProperty` with a dynamic getter.

Comment (phanect) [inline docs/docs/models.md]: Oh, OK. Then I will update it.

Comment (mickhansen) [inline docs/docs/models.md]: Great :)

**Comment by phanect** [comment]
@mickhansen Thanks for your comments ;) I've updated commits.

**Comment by mickhansen** [comment]
Great, thank you @phanect

**Pull request merged**